src/components/interaction-designer/block-editors/PhotoPromptEditor.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div class="photo-prompt-editor">
+    <label class="text-primary">{{ 'flow-builder.photo-response-prompt' | trans }}</label>
+    <div class="custom-control custom-checkbox">
+      <input
+        id="shouldIncludePrompt"
+        :value="shouldSetContactProperty"
+        :checked="shouldSetContactProperty"
+        type="checkbox"
+        name="includePrompt"
+        class="custom-control-input"
+        @change="toggleIncludePrompt">
+      <label
+        class="custom-control-label font-weight-normal"
+        for="shouldIncludePrompt">
+        {{ 'flow-builder.include-photo-response-prompt' | trans }}
+      </label>
+    </div>
+    <resource-editor
+      v-if="promptResource"
+      :resource="promptResource"
+      :block="block"
+      :flow="flow" />
+  </div>
+</template>
+
+<script lang="ts">
+import {Component, Prop} from 'vue-property-decorator'
+import {mixins} from 'vue-class-component'
+import Lang from '@/lib/filters/lang'
+import {IBlock, IFlow, IResource} from '@floip/flow-runner'
+import {namespace} from 'vuex-class'
+import ResourceEditor from '../resource-editors/ResourceEditor.vue'
+
+const flowVuexNamespace = namespace('flow')
+
+@Component({
+  components: {
+    ResourceEditor,
+  },
+})
+class LabelEditor extends mixins(Lang) {
+  @Prop() readonly block!: IBlock
+  @Prop() readonly flow!: IFlow
+
+  shouldSetContactProperty = false
```

Comment:
I don't really understand why this is called `shouldSetContactProperty`. Can you explain?

Reply:
Very nice catch @jcbashdown , I made some logic copy-paste from other component. It should have been `shouldIncludePrompt`. 
I made a fix, thanks